test/integration_test.py

```diff
@@ -439,18 +439,21 @@ def _check_terra_bdbag(self, catalog: CatalogName, response: bytes):
                  drs_uri, name, catalog, size)
         plugin = self.azul_client.repository_plugin(catalog)
         drs_client = plugin.drs_client()
-        access = drs_client.get_object(drs_uri, access_method=AccessMethod.https)
+        access = drs_client.get_object(drs_uri, access_method=AccessMethod.gs)
+        # TDR quirkily uses the GS access method to provide both a
+        # GS access URL *and* an access ID that produces an HTTPS
+        # signed URL
+        #
+        # https://github.com/ga4gh/data-repository-service-schemas/issues/360
+        # https://github.com/ga4gh/data-repository-service-schemas/issues/361
```

Comment:
Not sure this comment needs to be repeated here. It can be found in `DRSClient._get_object`, and the assert below tells us a `https` url is expected.

Reply:
Hannes requested that it be duplicated while [reviewing the previous PR](https://github.com/DataBiosphere/azul/pull/3245#discussion_r684535179)